app/controllers/api/reports_controller.rb

```diff
@@ -1,21 +1,21 @@
 class Api::ReportsController < Api::BaseController
 
   def index
-    { :entries => Report.all }
+    Report.all
```

Comment:
Let's leave wrapping for the collection resource (`entries` is fine), because if we return a JSON array, there is no place to add `_links` (e.g. for paging) or `_embedded`